jenkins/diff-job-config-patch.sh

```diff
@@ -66,13 +61,8 @@ docker run --rm=true -i \
 
 docker run --rm=true -i \
   -v "${docker_volume_output_dir}/patch:/output" \
-  -v "${REPO_DIR}:/kubernetes:ro" \
+  -v "${REPO_DIR}:/test-infra:ro" \
   "${JOB_BUILDER_IMAGE}" \
   bash -c "${common_commands}"
 
-diff -ruN "${output_dir}/upstream" "${output_dir}/patch" >"${diff_file}" || true
-if [[ -t 1 ]]; then  # Attached to a terminal?
-  less "${diff_file}"
-fi
-echo
-echo "  *** Diff saved in ${diff_file} ***"
+diff -ruN "${output_dir}/upstream" "${output_dir}/patch" || true
```

Comment:
I kinda like having a pager. Maybe use `less -F` if interactive and `${CI} !=true`, `cat` otherwise?